test-it/karma/fixtures/javascriptinvoke/ReinitializeToDefault.via

```diff
@@ -0,0 +1,22 @@
+define (MyVI dv(.VirtualInstrument (
+    Locals: c(   // Data Space
+        ce(dv(.Double 380)c1)
+        e(dv(.Double 1280)dataItem_Numeric)
```

Comment:
As mentioned in the conversation, please add more test cases for different data types (controls). Would like to take a look at it with more tests.